examples/graphene_apps/python_worker/avalon_worker/work_order_processor.py

```diff
@@ -0,0 +1,364 @@
+#!/usr/bin/python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import os
+import logging
+import socket
+import base64
+import random
+import secrets
+import string
+import json
+from hashlib import sha256
+import avalon_worker.receive_request as receive_request
+import avalon_worker.worker_encryption as worker_encryption
+import avalon_worker.worker_signing as worker_signing
+import avalon_worker.worker_hash as worker_hash
+import avalon_worker.workload_processor as workload_processor
+from avalon_worker.error_code import WorkerError
+import avalon_worker.utility.jrpc_utility as jrpc_utility
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+# -------------------------------------------------------------------------
+
+
+def main(args=None):
+    """
+    Graphene worker main function
+    """
+    # Create Process work order object
+    wo_processor = WorkOrderProcessor()
+    # Setup ZMQ channel to receive work order
+    url = "tcp://*:7777"
```

Comment:
Should this not go into a config file  or cmdline arg atleast ? And *, is that correct? Or is it that since this is just a POC code, it is fine?

Reply:
Yes. config file will be better. But as of now I'll have only this one in Python worker config file. I'll try to move this to config file